src/groups/mqb/mqbblp/mqbblp_cluster.cpp

```diff
@@ -1588,19 +1571,14 @@ bool Cluster::validateMessage(mqbi::QueueHandle**       queueHandle,
             subQueueIt == queueState.d_subQueueInfosMap.end())) {
         BSLS_PERFORMANCEHINT_UNLIKELY_HINT;
 
-        *errorStream << "message for unknown queue";
-
-        return false;  // RETURN
+        return k_UNKNOWN_QUEUE;  // RETURN
```

Comment:
Not strictly necessary, but we might want to distinguish between unknown queue (line 1561) and unknown subqueue (here)